man/crystal.1

```diff
@@ -496,6 +496,10 @@ Low optimization
 Middle optimization
 .It Fl O3
 High optimization
+.It Fl Os
```

Comment:
For some reason `-Os` shows up as `-  ` when I run `man -l man/crystal.1` while the other options show up correctly :confused:

Reply:
I ran `man` on MacOS against this file and it worked fine (see screenshot).

I'm wondering if `Os` might be getting interpreted as a request or a macro, since I see it documented as both.
If so, perhaps it needs escaping or enquoting?  I enquoted it locally and it continued to work.

```suggestion
.It Fl "Os"
```

Reply:
Nope, quotes have no effect. I'm on Ubuntu Linux 20.04:

```
     -O0       No optimization (default)
     -O1       Low optimization
     -O2       Middle optimization
     -O3       High optimization
     -         Middle optimization with focus on file size
     -Oz       Middle optimization aggressively focused on file size
```

Reply:
It's indeed a thing in troff: https://www.gnu.org/software/groff/manual/groff.html#index-os

Reply:
Fixed with an explicit escape sequence `\!Os` so it's not interpreted as the `.os` request.